app/authentication/bearer_auth.py

```diff
@@ -0,0 +1,14 @@
+import requests
+
+__all__ = [
+    'BearerAuth'
+]
```

Comment:
Is this needed? Other classes in our project don't define that, what would be the benefit? Don't think we ever `from .. import *`

Reply:
This is good practice to declare public and internal interfaces regardless of import *
You can see it even made it into the pep-0008 guidance: 
https://www.python.org/dev/peps/pep-0008/#public-and-internal-interfaces